manifests/d/DarkfullDante/srtAlign/0.6.1/DarkfullDante.srtAlign.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: "DarkfullDante.srtAlign"
+PackageVersion: "0.6.1"
+PackageLocale: "en-US"
+Publisher: "DarkfullDante"
+PackageName: "SRT Align"
```

```suggestion
PackageName: "srt-align"
```
didn't want to do this but it needs to match ARP...

---

Accepted the change, my bad, misunderstood the line in the documentation

---

Yeeeeh its common in the start, in short; `PackageName`, `Publisher` and `PackageVersion` needs to match what control panel sees, except it accepts a wildcard at the end, so instead of for example `SRT Align 0.6.1` you can do `SRT Align` then WinGet looks for `SRT Align *` (in your case it got a dumb - in the middle)